bibliopixel/layout/layout.py

```diff
@@ -60,7 +61,7 @@ def color_list(self):
     def _get_base(self, pixel):
         if pixel >= 0 and pixel < self.numLEDs:
             return self._colors[pixel]
-        return 0, 0, 0  # don't go out of bounds
+        return self._empty  # don't go out of bounds
 
     def _set_base(self, pixel, color):
         if pixel >= 0 and pixel < self.numLEDs:
```

Comment:
Note... this will need to handle things correctly if using an RGBW color space and someone passes colors.Red which only has 3 values.